### PR TITLE
Men 1777 rocko

### PR DESCRIPTION
**Pull request by alfrunes**
@kacf @drewmoseley

### CODE REVIEW

**Comment by drewmoseley** [comment]
Are you sure https://github.com/mendersoftware/meta-mender/pull/554/commits/28ea9c68bded3ccc0ee0680ec91785227a6e571a is applicable to rocko?

I was just starting to look at most of these patches as part of my rocko branch maintenance effort.  What is your intent here? Are all of these needed to support your work or should we split this into new stuff and backports?

File: meta-mender-toradex-nxp/conf/machine/colibri-imx7-mender.conf [hunk 1]
```diff
@@ -56,3 +60,7 @@ MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += "mtd-utils mtd-utils-ubifs"
 # pick u-boot-toradex-fsl-fw-utils as provider of fw_{printenv,setenv}
 PREFERRED_PROVIDER_u-boot-fw-utils ?= "u-boot-toradex-fsl-fw-utils"
 PREFERRED_RPROVIDER_u-boot-fw-utils ?= "u-boot-toradex-fsl-fw-utils"
+
+# Set product id vars (needed by toradex_image_json)
+TORADEX_PRODUCT_IDS_append = " 0033"
+TORADEX_PRODUCT_IDS[0033] = "colibri-imx7"
```
Comment (kacf) [inline meta-mender-toradex-nxp/conf/machine/colibri-imx7-mender.conf]: Should this also use `?=`?

File: meta-mender-toradex-nxp/recipes-bsp/u-boot/patches/0001-Integration-of-Mender-boot-code-into-U-Boot-v2016.11.patch [hunk 2]
```diff
@@ -0,0 +1,49 @@
+From 2013899468926d9b7128314b6d2052105e6858ed Mon Sep 17 00:00:00 2001
```
Comment (kacf) [inline meta-mender-toradex-nxp/recipes-bsp/u-boot/patches/0001-Integration-of-Mender-boot-code-into-U-Boot-v2016.11.patch]: I can't see that you are referring to this patch file (and the next one) anywhere. Are they shadowing the patch files found in meta-mender-core, or did you forget to remove them?

File: meta-mender-toradex-nxp/conf/machine/colibri-imx7-mender.conf (hunk 1 above)
Comment (alfrunes) [reply meta-mender-toradex-nxp/conf/machine/colibri-imx7-mender.conf]: Is it legal to use default assignment (`?=`) when doing  an override append?

Comment (kacf) [reply meta-mender-toradex-nxp/conf/machine/colibri-imx7-mender.conf]: Ah, I think you're right, it isn't. Never mind then.

File: meta-mender-toradex-nxp/recipes-bsp/u-boot/patches/0001-Integration-of-Mender-boot-code-into-U-Boot-v2016.11.patch (hunk 2 above)
Comment (alfrunes) [reply meta-mender-toradex-nxp/recipes-bsp/u-boot/patches/0001-Integration-of-Mender-boot-code-into-U-Boot-v2016.11.patch]: I can't figure out where these patches came from, so I removed them, rebased and repushed.

**Comment by drewmoseley** [comment]
Looks good.  Merging.

**Pull request merged**